rllib/env/multi_agent_env.py

```diff
@@ -49,9 +49,15 @@ def __init__(self):
             self._spaces_in_preferred_format = None
 
     @PublicAPI
-    def reset(self) -> MultiAgentDict:
+    def reset(
+        self,
+        seed: Optional[int] = None,
+    ) -> Tuple[MultiAgentDict, MultiAgentDict]:
```

Comment:
Doesn't reset also now support reset kwargs? We would need to add those if that's the case

Reply:
Your were right, this was added as `options`, just like in gymnasium. It's currently NOT supported for RLlib users (won't break, but users cannot add any content per episode to that kwargs dict).

I wanted to enable this in a follow-up PR.